libraries/AP_Scripting/examples/esc-usage.lua

```diff
@@ -0,0 +1,15 @@
+-- This script displays ESC usage time (supported only by ToshibaCAN ESCs)
+
+function update() -- this is the loop which periodically runs
+  if not arming:is_armed() then -- only run check when disarmed
+    local got_usage = false
```

Comment:
unused?

Reply:
oops.  I've added a "gcs:send_text("ESC usage could not be retrieved")" if it failed to read any usage info from the ESCs